style.css

```diff
@@ -1,489 +1,772 @@
 * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-    --gold-color: #d2b429;
-    --header-font: 'Cormorant Upright', 'Georgia', serif;
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	--gold-color: #d2b429;
+	--header-font: "Old Standard TT", "Georgia", serif;
 }
 
 body {
-    background-color: black;
-    margin: 0;
-    font-family: 'Montserrat', sans-serif;
-    font-weight: 400;
-    font-size: 20px;
-    color: white;
+	background-color: black;
+	margin: 0;
+	font-family: "Montserrat", sans-serif;
+	font-weight: 400;
+	font-size: 20px;
+	color: white;
 }
 
 .clear {
-    clear: both;
+	clear: both;
 }
 
 .text-black {
-    color: #000;
-    line-height: 2.1rem;
+	color: #000;
+	line-height: 2.1rem;
 }
 
 p.text-white {
-    color: #fff;
-    line-height: 2.1rem;
-    padding-bottom: 2rem;
+	color: #fff;
+	line-height: 2.1rem;
+	padding-bottom: 2rem;
 }
 
 .uppercase {
-    text-transform: uppercase;
+	text-transform: uppercase;
 }
 
 button {
-    background-color: #fff;
-    color: var(--gold-color);
-    font-family: var(--header-font);
-    text-align: center;
-    border: solid 1px var(--gold-color);
-    font-size: 2rem;
+	background-color: #fff;
+	color: var(--gold-color);
+	font-family: var(--header-font);
+	text-align: center;
+	border: solid 1px var(--gold-color);
+	font-size: 2rem;
 }
 
 p.button-text {
-    padding: 1rem;
+	padding: 1rem;
 }
 
 .section-row {
-    max-width: 71rem;
-    min-height: 130px;
-    display: flex;
-    position: relative;
-    margin: auto;
+	max-width: 71rem;
```

Comment:
Some people would complain about "magic numbers", however I'm fine with using them. You may consider commenting the reason for using any "strange" numbers, or move the to variables.

Reply:
I don't have any idea how can I set max-with of the section in a different way. Would it be less magic if I set 70rem?

Reply:
Haha, actually yes. :) It's one of the stiff patterns that may be loosened up a bit but it may happen in the future that if someone encountered a number in your code that is not round and not obvious, they'd ask for an explanation in a form of a little comment or a well-named variable. :)

Reply:
I've replaced it with var.